src/test/java/duke/ResponseTest.java

```diff
@@ -0,0 +1,18 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ResponseTest {
+    private static String LINE = "    ____________________________________________________________";
+    private static String MESSAGE_INDENT = "     ";

```

Comment:
You can use the \t escape sequence in your string to provide tab spaces. However, if you were planning to customise the amount of tab space then this works too!